initializr-generator/src/main/resources/templates/kubernetes/kubernetes.yaml

```diff
@@ -2,42 +2,43 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{name}}
+  name: {{k8sName}}
   labels:
-    run: {{name}}
+    run: {{k8sName}}
 spec:
   ports:
   - port: {{port}}
     protocol: TCP
   selector:
-    run: {{name}}
+    run: {{k8sName}}
+  type: LoadBalancer
 
 ---
 
 apiVersion: apps/v1
 kind: Deployment
```

Comment:
shoud it be apiVersion: extensions/v1beta1? I came across this error:
Error from server (BadRequest): error when creating "kubernetes/kubernetes.yaml": Deployment in version "v1" cannot be handled as a Deployment: no kind "Deployment" is registered for version "apps/v1"